node/cli-opt/src/account_key.rs

```diff
@@ -15,24 +15,24 @@
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
 use bip39::{Language, Mnemonic, MnemonicType, Seed};
+use clap::Parser;
 use libsecp256k1::{PublicKey, SecretKey};
 use primitive_types::H256;
 use sp_runtime::traits::IdentifyAccount;
-use structopt::StructOpt;
 use tiny_hderive::bip32::ExtendedPrivKey;
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 pub struct GenerateAccountKey {
 	/// Generate 12 words mnemonic instead of 24
-	#[structopt(long, short = "w")]
+	#[clap(long, short = 'w')]
```

Comment:
Why is it changing format for single/double quote ?

Reply:
Must be char literal instead of string literal now (change from structopt to clap v3)